src/components/grid-aware/Spacer/Spacer.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import s from "./Spacer.module.css";
+
+const Spacer = ({ heightDesktop, heightMobile, color }) => (
+  <div
+    style={{
+      "--spacer-height-desktop": heightDesktop,
+      "--spacer-height-mobile": heightMobile,
+      "--spacer-color": `${color}`,
```

Comment:
This one also doesn't require the string interpolation, and you can just directly pass in the variable.

```suggestion
      "--spacer-color": color,
```